opm/material/fluidsystems/blackoilpvt/ConstantCompressibilityWaterPvt.hpp

```diff
@@ -209,6 +209,31 @@ class ConstantCompressibilityWaterPvt
         return (1.0 + X*(1.0 + X/2.0))/BwRef;
     }
 
+
+    template <class Evaluation>
+    void inverseBAndMu(Evaluation& bw, Evaluation& muW, unsigned regionIdx,
+                      // const Evaluation& /*temperature*/,
+                      const Evaluation& pressure) const
+                      // const Evaluation& /*Rsw*/,
+                      // const Evaluation& /*saltconcentration*/) const
+    {
+        // cf. ECLiPSE 2011 technical description, p. 116
```

Comment:
This is not an published document, write explanation and/or formula instead.

Reply:
removed